cartographer/mapping/3d/submap_3d.h

```diff
@@ -97,14 +102,18 @@ class ActiveSubmaps3D {
   // Inserts 'range_data_in_local' into the Submap collection.
   // 'local_from_gravity_aligned' is used for the orientation of new submaps so
   // that the z axis approximately aligns with gravity.
-  std::vector<std::shared_ptr<const Submap3D>> InsertRangeData(
+  // 'rotational_scan_matcher_histogram_in_local' will be accumulated in all
```

Comment:
This comment needs to be fixed: `rotational_scan_matcher_histogram_in_gravity`

Reply:
done